README

```diff
@@ -9,13 +9,13 @@ environment (parsing and transforming libegalito.so).
 
 Egalito supports x86_64 and aarch64, with experimental support for RISC-V.
 
-For more information, please visit https://egalito.org .
+For more information, please visit https://egalito.org.
```

Comment:
this space is so that double-click & copy works in a terminal :)

Reply:
c'mon ...